tests/losses/test_ssl.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import torch
+
+from torchgeo.losses import NTXentLoss
+
+
+class TestSSLLosses:
+    def test_ntxent_loss(self) -> None:
+        z1 = torch.randn(2, 8)
+        z2 = torch.randn(2, 8)
+        t = torch.tensor(0.5)
+        loss = NTXentLoss()(z1, z2, t)
+        assert loss.ndim == 0
+        assert loss.dtype == torch.float
```

Comment:
We should test the value of `loss` to make sure it is correct, not just that it runs without crashing. This would require using a manually defined matrix, not just a random tensor.

Reply:
Is it sufficient to insert z1 and z2 where each are just vectors with 1s in them and compute the loss and compare to this? Or is there a specific case you would like to see?

Reply:
I would just choose some simple vectors (maybe [0, 1, 2] and [3, 4, 5] or something of that level of complexity), then manually compute what NT-Xent should be using a mathematical formula, then confirm that the output of your loss matches.